lib/studies/reply/adaptive_nav.dart

```diff
@@ -1094,17 +1098,25 @@ class _ReplyFabState extends State<_ReplyFab>
       openColor: theme.cardColor,
       closedShape: isDesktop & widget.extended
           ? const StadiumBorder()
-          : const CircleBorder(),
+          : isDesktop ? const CircleBorder() : mobileFabBorder,
```

Comment:
How come you can't continue to use `CircleBorder`?

Reply:
You're right should've tried that first.